pkg/report/report.go

```diff
@@ -425,9 +442,40 @@ func Generate(ctx context.Context, path string, mrs yara.MatchRules, c malconten
 				syscalls = append(syscalls, sy...)
 			case "cap":
 				caps = append(caps, v)
+			// Reduce the severity by one level if the path contains `v`
+			case "decrease_if":
+				if strings.Contains(path, v) {
```

Comment:
For both `decrease_if` and `drop_if`, we could support comma-delimited metadata values. 

For the former, we could set the severity to drop to and then the binary/path, e.g., `low,bash`.

For the latter, we could specify a list of binaries, e.g., `bash,fish,sh`.